predator/README.md

```diff
@@ -35,8 +35,12 @@ The command removes all the Kubernetes components associated with the chart and
 
 ## Configuration
 
-By default `Predator` runs in with sqlite as it's storage, so it doesn't actually persist anything other than in memory.
-`Predator` supports other storage backends like Cassandra, postgresSQL, MySql and SQL Server.
+`Predator` with SQLite persisted storage 
```

Comment:
include above: 
`By default configuration `Predator` runs in with sqlite as it's storage without persistence.`

It is possible to run with persistence with the following: